cinema/tests/test_movie_api.py

```diff
@@ -114,16 +112,12 @@ def test_put_movie(self):
         }
 
         response = self.client.put(url, payload)
-        self.assertEqual(
-            response.status_code, status.HTTP_405_METHOD_NOT_ALLOWED
-        )
+        self.assertEqual(response.status_code, status.HTTP_405_METHOD_NOT_ALLOWED)
```

Comment:
The `test_put_movie` method expects a `405 Method Not Allowed` response. Ensure that the API is configured to disallow PUT requests on the movie detail endpoint if this is the expected behavior. Otherwise, adjust the test to match the actual API behavior.